### DISCUSSION TITLE
Unable to Submit Support Ticket Due to SMS Verification Issues with +86 Numbers

### DISCUSSION THREAD

**Discussion by Azir-11** [community/community#145185] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have already enabled 2FA, but when submitting a support ticket, it still requires SMS verification. However, it does not support Chinese +86 phone numbers. Additionally, emails sent to support@github.com are being blocked, which leaves me with no way to report or resolve the issue of my account being flagged. What should I do?

**Comment by cng10001** [comment]
楼主最后是如何解决的呢？
